enterprise/frontend/src/embedding-sdk/cli/steps/check-sdk-available.ts

```diff
@@ -0,0 +1,27 @@
+import ora from "ora";
+
+import { SDK_PACKAGE_NAME } from "embedding-sdk/cli/constants/config";
+import { installSdk } from "embedding-sdk/cli/steps/install-sdk";
+import { getPackageVersion } from "embedding-sdk/cli/utils/get-package-version";
+
+import type { CliStepMethod } from "../types/cli";
+
+export const checkSdkAvailable: CliStepMethod = async state => {
+  const spinner = ora("Checking if SDK is installed…").start();
+  const sdkDep = await getPackageVersion(SDK_PACKAGE_NAME);
```

Comment:
```suggestion
  const sdkVersion = await getPackageVersion(SDK_PACKAGE_NAME);
```

Would this be more correct?